### ISSUE TITLE
Requesting formal literature or mathematical equations for GradientSHAP

### ISSUE THREAD

**Issue by kountaydwivedi** [pytorch/captum#1040]
## 📚 Documentation

@NarineK and others..

Dear developers,
As a part of a research project, I need to explain the processing of GradientSHAP in a mathematical format. However, so far, I have not been able to find any literature providing extensive explanation or any mathematical background, and the manual provided in the documentation is ambiguous and not so clear.

It is a sincere request to kindly lay down some mathematical equations/approach, so that implemention of the method _GradientSHAP_ could become better understandable.

Thank you.
Looking forward to your reply.

Kountay Dwivedi

**Comment by NarineK** [comment]
@kountaydwivedi, I think that the closed is the Expected Gradients also described here: https://arxiv.org/pdf/1906.10670.pdf

**Comment by kountaydwivedi** [comment]
> @kountaydwivedi, I think that the closed is the Expected Gradients also described here: https://arxiv.org/pdf/1906.10670.pdf https://github.com/slundberg/shap#deep-learning-example-with-gradientexplainer-tensorflowkeraspytorch-models

Thank you so much Ma'am for your efforts in providing this insightful article.

**Comment by kountaydwivedi** [comment]
> @kountaydwivedi, I think that the closed is the Expected Gradients also described here: https://arxiv.org/pdf/1906.10670.pdf https://github.com/slundberg/shap#deep-learning-example-with-gradientexplainer-tensorflowkeraspytorch-models

**Comment by kountaydwivedi** [comment]
@NarineK 
Thank you so much Ma'am for your efforts in providing this insightful article.
